tests/gnrc_tcp/tests/test_01-conn_lifecycle_as_client.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018   Simon Brummer <simon.brummer@posteo.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+import socket
+import threading
+from shared_func import getHostLinkLocalAddr
+from shared_func import getRiotInterfaceId
+
+HOST_IF = 'tap0'
+SERVER_PORT = 15000
+
+def tcpServer():
+    sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+    sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    sock.bind(('::', SERVER_PORT))
+    sock.listen(1)
+
+    conn, addr = sock.accept()
+    conn.close()
+
+def testfunc(child):
+    serverHandle = threading.Thread(target = tcpServer)
+    serverHandle.start()
+
+    targetAddr = getHostLinkLocalAddr(HOST_IF) + '%' + getRiotInterfaceId(child)
+
+    # Setup RIOT Node to connect to Hostsystems TCP Server
+    child.sendline('gnrc_tcp_tcb_init')
+    child.sendline('gnrc_tcp_open_active AF_INET6 ' + targetAddr + ' ' + str(SERVER_PORT) + ' 0')
+    child.expect_exact('gnrc_tcp_open_active: returns 0')
+
+    # Close connection
+    child.sendline('gnrc_tcp_close')
+    serverHandle.join()
+
+if __name__ == '__main__':
+    sys.path.append(os.path.join(os.environ['RIOTTOOLS'], 'testrunner'))
+    import testrunner
+
+    scriptName = os.path.basename(sys.argv[0]);
+    res = 1
+
+    try:
+        res = testrunner.run(testfunc, timeout = 3, echo = False, traceback = False)
```

Comment:
Why not just exit as all the other scripts to

Reply:
Also `pep8`: `timeout=3, echo=False, ...` for parameter assignments

Reply:
Also also: why deactivate traceback? It's really helpful to find the line that failed ;-)

Reply:
Done